src/main/java/com/hobbycollection/dao/CollectionDAO.java

```diff
@@ -9,6 +9,8 @@
 
 public class CollectionDAO implements ICollectionDAO{
 
+    //This is how the endpoints interact with the data.
+
```

Comment:
1) Use proper JavaDoc format above a method.
2) It's a given that DAOs do this in a Spring Boot project, so this comment does not add much value.